dist/Dockerfile.e2e-ubi9/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM registry.access.redhat.com/ubi9/ubi:latest as rust_builder
```

Comment:
Here, we're creating a new dockerfile for ubi9. 
we should just move away from ubi8 to ubi9. 
Lets replace the existing ubi8. 
Also, we'll still be testing with ubi8 using the rust-toolset dockerfile

Reply:
Replaced ubi8 with ubi9 and renamed the file.
It will cause a short period of broken CI before https://github.com/openshift/release/pull/62510 gets in